feat/time-capsule-detail/src/main/java/com/emotionstorage/time_capsule_detail/ui/component/SaveTimeCapsuleButton.kt

```
@@ -0,0 +1,113 @@
+            isDefaultWidth = false,
+            isDefaultHeight = false,
+        ) {
+            CountDownTimer(
+                deadline = expireAt ?: LocalDateTime.now().plusHours(24),
+            ) { hours, minutes, seconds ->
+                LaunchedEffect(hours, minutes, seconds) {
+                    if (hours == 0L && minutes == 0L && seconds == 0L) {
+                        onExpire()
+                    }
+                }
+                val remainingTime = String.format("%02d:%02d:%02d", hours, minutes, seconds)
```

detekt (1.23.8)

[warning] String.format("%02d:%02d:%02d", hours, minutes, seconds) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)